docker-compose-earthly.yml

```
@@ -9,6 +9,7 @@ services:
       - KUBERPULT_DEX_MOCK=false
       - KUBERPULT_DEX_ENABLED=false
       - KUBERPULT_GIT_WRITE_COMMIT_DATA=true
+      - KUBERPULT_ALLOW_LONG_APP_NAMES=false
```

```suggestion
      - KUBERPULT_ALLOW_LONG_APP_NAMES=true
```
generally, all feature flags should be enable for local development.

---

updated